td-payload/src/main.rs

```diff
@@ -34,7 +44,95 @@ extern "win64" {
     fn cet_ss_test(count: usize);
 }
 
-#[cfg(not(test))]
+pub struct PayloadHobInfo<'a> {
+    hob_list: &'a [u8],
+    page_table: pi::hob::MemoryAllocationHeader,
+    payload: pi::hob::MemoryAllocationHeader,
+    runtime: pi::hob::MemoryAllocationHeader,
```

Comment:
Do we assume only one runtime memory block is available?

Reply:
The 'runtime' indicates the whole region located at the highest memory below 4G which contains heap, stack, evenlog and ACPI table etc., in runtime layout.

Do you think it is good to name this block as highmem or reserved?

Reply:
why we need one specific block to indicate highmem?

td-payload cannot make any assumption on td-shim, if it is not defined in td-shim spec.